--- wxWidgets-3.0.0/src/gtk/app.cpp~	2013-11-11 14:10:39.000000000 +0100
+++ wxWidgets-3.0.0/src/gtk/app.cpp	2014-01-19 16:44:33.612093277 +0100
@@ -168,7 +168,7 @@
 // Access to the root window global
 //-----------------------------------------------------------------------------
 
-GtkWidget* wxGetRootWindow()
+WXDLLIMPEXP_CORE GtkWidget* wxGetRootWindow()
 {
     static GtkWidget *s_RootWindow = NULL;
 
