--- wxWidgets-2.6.1/configure.in.orig	2005-09-04 17:57:44.359484848 +0200
+++ wxWidgets-2.6.1/configure.in	2005-09-04 17:58:34.275896400 +0200
@@ -2608,6 +2608,8 @@
     WIN32INSTALL=win32install
 fi
 
+PKG_PROG_PKG_CONFIG
+
 if test "$wxUSE_GUI" = "yes"; then
     USE_GUI=1
 
@@ -2721,10 +2723,16 @@
             save_CFLAGS="$CFLAGS"
             save_LIBS="$LIBS"
             CFLAGS="$CFLAGS $wx_cv_cflags_gtk"
-            LIBS="$LIBS $wx_cv_lib_gtk"
+            LIBS="$LIBS $wx_cv_libs_gtk"
             
             AC_CHECK_FUNCS([pango_font_family_is_monospace])
             
+            dnl needed for pango_x_get_context() in src/gtk/window.cpp
+            PKG_CHECK_MODULES(PANGOX, pangox,
+                    [wx_cv_libs_gtk="$wx_cv_libs_gtk $PANGOX_LIBS"],
+                    [AC_MSG_ERROR([pangox library not found])]
+            )
+
             dnl test if we have at least GTK+ 2.4:
             AC_MSG_CHECKING([if GTK+ is version >= 2.4])
             AC_TRY_COMPILE([
