--- wxWidgets-2.8.12/contrib/include/wx/mmedia/vidbase.h.orig	2011-03-22 13:01:05.000000000 +0100
+++ wxWidgets-2.8.12/contrib/include/wx/mmedia/vidbase.h	2012-10-03 19:04:15.935793663 +0200
@@ -16,7 +16,6 @@
 // headers
 // ----------------------------------------------------------------------------
 // For compilers that support precompilation, includes "wx/wx.h".
-#include "wx/wxprec.h"
 
 #ifdef __BORLANDC__
     #pragma hdrstop
@@ -24,13 +23,11 @@
 
 // for all others, include the necessary headers (this file is usually all you
 // need because it includes almost all "standard" wxWidgets headers
-#ifndef WX_PRECOMP
     #include "wx/defs.h"
     #include "wx/stream.h"
     #include "wx/string.h"
     #include "wx/window.h"
     #include "wx/frame.h"
-#endif
 
 #include "wx/mmedia/defs.h"
 
--- wxWidgets-2.8.12/contrib/include/wx/mmedia/vidxanm.h.orig	2011-03-22 13:01:05.000000000 +0100
+++ wxWidgets-2.8.12/contrib/include/wx/mmedia/vidxanm.h	2012-10-03 19:05:04.928013219 +0200
@@ -15,7 +15,6 @@
 // headers
 // ----------------------------------------------------------------------------
 // For compilers that support precompilation, includes "wx/wx.h".
-#include "wx/wxprec.h"
 
 #ifdef __BORLANDC__
     #pragma hdrstop
@@ -25,10 +24,8 @@
 
 // for all others, include the necessary headers (this file is usually all you
 // need because it includes almost all "standard" wxWidgets headers
-#ifndef WX_PRECOMP
     #include "wx/defs.h"
     #include "wx/string.h"
-#endif
 
 // ----------------------------------------------------------------------------
 // System dependent headers
