--- wxWidgets-2.5.3/contrib/build/ogl/ogl.bkl.wiget	2005-01-21 15:50:32.000000000 +0100
+++ wxWidgets-2.5.3/contrib/build/ogl/ogl.bkl	2005-01-21 15:51:15.000000000 +0100
@@ -43,7 +43,8 @@
     <dll id="ogldll" template="wx_contrib_dll" cond="SHARED=='1'">
         <define>WXUSINGDLL</define>
         <define>WXMAKINGDLL_OGL</define>
-        <sources>$(OGL_SRC)</sources>
+		<sources>$(OGL_SRC)</sources>
+		<wx-lib>deprecated</wx-lib>
         <wx-lib>core</wx-lib>
         <wx-lib>base</wx-lib>
     </dll>
