diff -uNr wxWidgets-2.8.1.orig/src/common/pickerbase.cpp wxWidgets-2.8.1/src/common/pickerbase.cpp
--- wxWidgets-2.8.1.orig/src/common/pickerbase.cpp	2007-01-08 15:16:57.000000000 +0100
+++ wxWidgets-2.8.1/src/common/pickerbase.cpp	2007-01-14 16:51:57.000000000 +0100
@@ -124,7 +124,9 @@
     SetMinSize( m_sizer->GetMinSize() );
 }
 
-void wxPickerBase::DoSetToolTip( wxToolTip *tip )
+#if wxUSE_TOOLTIPS
+
+void wxPickerBase::DoSetToolTip(wxToolTip *tip)
 {
     // don't set the tooltip on us but rather on our two child windows
     // as otherwise it would appear only when the cursor is placed on the
@@ -135,7 +137,7 @@
     m_text->SetToolTip(tip ? new wxToolTip(tip->GetTip()) : NULL);
 }
 
-
+#endif // wxUSE_TOOLTIPS
 
 // ----------------------------------------------------------------------------
 // wxPickerBase - event handlers
