--- wxWidgets-3.1.5/src/html/m_image.cpp~	2021-04-12 22:23:58.000000000 +0000
+++ wxWidgets-3.1.5/src/html/m_image.cpp	2021-11-30 11:33:49.910935776 +0000
@@ -409,8 +409,8 @@
                         {
                             m_gifTimer = new wxGIFTimer(this);
                             long delay = m_gifDecoder->GetDelay(0);
-                            if ( delay == 0 )
-                                delay = 1;
+                            if ( delay < 10 )
+                                delay = 10;
                             m_gifTimer->Start(delay, true);
                         }
                         else
@@ -532,8 +532,8 @@
     }
 
     long delay = m_gifDecoder->GetDelay(m_nCurrFrame);
-    if ( delay == 0 )
-        delay = 1;
+    if ( delay < 10 )
+        delay = 10;
     timer->Start(delay, true);
 }
 #endif
